Tasks/Barys/Task3_1/query07.sql

```diff
@@ -0,0 +1,16 @@
+--Query 7
+SELECT DISTINCT
+    YEAR(SOH.OrderDate) AS Year,
+    SOH.SalesOrderID,
+    SOH.CustomerID,
+    SOH.TotalDue,
+    SUM(SOH.TotalDue) OVER() AS TotalBusinessSales,
+    SUM(SOH.TotalDue) OVER(PARTITION BY YEAR(SOH.OrderDate)) AS CustomerSalesOverYear,
```

Comment:
while there is no group by it all customer sales over year and not all sales per customer per year (probably task desc is not clear)